src/components/Carts/cart.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import useCartHooks from "../../redux/Cart/cart-actions";
+import { Col, Form } from 'react-bootstrap';
+import {ButtonStyled as Button , CartStyled} from './cart.style'
+import  {ProductCardFtr} from '../ProductDetails/details.style';
+
+const CartItem = ({ item }) => {
+  const { _id, image, title, description, price ,qty } = item;
+  const [input, setInput] = useState(qty);
+  const { adjustItemQty, removeFromCart } = useCartHooks();
+  
+
+  const onChangeHandler = (e) => {
+    const { value } = e.target;
+    setInput(value);
+    adjustItemQty(_id, value);
+  };
```

Comment:
No need to setState here, if not using value anywhere
```suggestion
  const onChangeHandler = (e) => {
    const { value } = e.target;
    adjustItemQty(_id, value);
  };
```